libs/cluster/Session/RespClusterFailoverCommands.cs

```diff
@@ -43,14 +40,12 @@ private bool NetworkClusterFailover(int count, out bool invalidParameters)
                     failoverOption = FailoverOption.INVALID;
                 }
 
-                if (count > 1)
+                if (parseState.Count > 1)
                 {
-                    if (!RespReadUtils.ReadIntWithLengthHeader(out var failoverTimeoutSeconds, ref ptr, recvBufferPtr + bytesRead))
-                        return false;
+                    var failoverTimeoutSeconds = parseState.GetInt(1);
```

Comment:
Should we be using TryGet for every numerical value?

Reply:
We definitely should! I fixed (I hope) all the explicit conversions in other parts of the code, I was just overwhelmed by the amount of them here, since I need to look up the correct error response for each of these occurrences... Another thing to do is to add tests for erroneous inputs for each command...

Reply:
Ok ended up not being as bad as I thought - done! :)